recipe/migrations/macos_min_version1011.yaml

```diff
@@ -0,0 +1,18 @@
+migrator_ts: 1587148342
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+
+#TODO: What version do we want?
+#  10.11 is just a suggestion...
+macos_min_version:             # [osx]
+  - 10.11                      # [osx]
```

Comment:
We should think hard on the min version here. We have packages at 10.12 already IIUIC.